larky/src/main/resources/vendor/cryptography/hazmat/backends/pycryptodome.star

```diff
@@ -722,6 +957,12 @@ def pycryptodome():
         return self._evp_pkey_to_private_key(evp_pkey)
 
     self.load_pem_private_key = load_pem_private_key
+
+    def load_pem_public_key(data):
+        if(re.search(r'-----BEGIN (EC|RSA) PRIVATE KEY-----', data)):
```

Comment:
@OmegaPointZero can you help me understand why this is necessary? why wouldn't we call load_pem_private_key? Also, it looks like this is not what the cryptography library does... https://github.com/pyca/cryptography/blob/main/src/cryptography/hazmat/backends/openssl/backend.py#L920-L960